src/elm/Cambiatus/Object/Community.elm

```diff
@@ -54,6 +54,16 @@ description =
     Object.selectionForField "String" "description" [] Decode.string
 
 
+hasActions : SelectionSet Bool Cambiatus.Object.Community
+hasActions =
+    Object.selectionForField "Bool" "hasActions" [] Decode.bool
+
+
+hasShop : SelectionSet Bool Cambiatus.Object.Community
+hasShop =
+    Object.selectionForField "Bool" "hasShop" [] Decode.bool
+
+
```

Comment:
all files from this folder are automatically generated from the backend API

Reply:
Re-checked it once again and it looks fine. These functions are generated by `elm-graphql` from the actual `master` branch of the backend. Since this feature is [already merged](https://github.com/cambiatus/backend/commit/e5a3123add5990fd3f58e59228722d83fc909bd7), then we have it in Elm GraphQL autogenerated code.

Reply:
Oh, I see. Silly me, I wrote those by hand on the features branch :p